Add last step information (#16)
fal-ai/dbt-cloud-action: README.md

          fal run --profiles-dir .

```

#### Getting the correct artifacts from dbt-cloud

> fal relies on the generated artifacts from a dbt step to get model status. dbt-cloud only makes available the artifacts generated by the **last** step ran.

Reviewer: ```suggestion
fal relies on the generated artifacts from a dbt run step to get model statuses. dbt-cloud only makes these artifacts available after the **last** step finished running.
```

In order to get the status information that you need for fal, make sure to run the step you are interested in **last**.

> For example, this dbt job will make available the `run_results.json` of `dbt docs generate`, which is probably not what you want fal to report about.

Reviewer: ```suggestion
For example, this dbt job will provide the `run_results.json` of `dbt docs generate`, which is probably not what you want fal to report about:
```
